crates/orchestrator/src/tests/config.rs

```diff
@@ -64,116 +93,347 @@ impl TestConfigBuilder {
             queue: None,
             storage: None,
             alerts: None,
+
+            database_node: None,
+            data_storage_node: None,
+            data_storage_client: None,
+            queue_node: None,
+            queue_client: None,
+            alert_client: None,
+            alert_node: None,
         }
     }
 
-    pub fn mock_da_client(mut self, da_client: Box<dyn DaClient>) -> TestConfigBuilder {
+    pub async fn testcontainer_s3_data_storage(mut self) -> TestConfigBuilder {
+        let (node, storage_client, client) = s3_testcontainer_setup().await;
+        self.data_storage_node = Some(node);
+        self.data_storage_client = Some(client);
+        self.storage = Some(storage_client);
+        self
+    }
+
+    pub async fn testcontainer_sqs_queue(mut self) -> TestConfigBuilder {
+        let (node, queue_client, client) = sqs_testcontainer_setup().await;
+        self.queue = Some(queue_client);
+        self.queue_client = Some(client);
+        self.queue_node = Some(node);
+        self
+    }
+
+    // IMP! Don't use SQS ans SNS testcontainer setups together
```

Comment:
ADDED : 
`SQS testcontainer setup creates all the queues in one go, except for the test queue for SNS`
`SNS testcontainer internally calls SQS testcontainer setup as well as creates the test queue for SNS, so if we have to use the other queues, rather than recreating them with SQS setup, we should utilise the ones already created with SNS`

Reply:
I would ideally do this in a way where SNS setup doesn't call SQS setup (as that creates a lot of other queues), instead I will just create the specific queue needed for SNS in the sns setup